explainer.md

```diff
@@ -0,0 +1,207 @@
+# Introduction
+Smooth visual transitions as users navigate on the web can lower cognitive load by helping users stay in context. It can also provide a [visual cue](https://www.wired.com/2014/04/swipe-safari-ios-7/) about the destination before initiating the navigation. Both site authors and UAs add visual transitions to their navigations for these use-cases.
+
+However, the user experience is bad if [both the site author and the UA](https://stackoverflow.com/questions/19662434/in-ios-7-safari-how-do-you-differentiate-popstate-events-via-edge-swipe-vs-the) add these transitions. The goal of this proposal is to avoid such cases to ensure only one visual transition is executed at a time.
+
+The ethos guiding the API design are:
+* If a visual transition can be defined by both the site author and the UA, the site author takes precendence.
```

Comment:
```suggestion
* If a visual transition can be defined by both the site author and the UA, the site author takes precendence, since they have a better understanding of the design and functionality of their site.
```